organisaatio-ui/src/components/Sivut/LomakeSivu/LomakeSivu.tsx

```diff
@@ -277,11 +287,32 @@ const LomakeSivu = ({ match: { params }, history }: LomakeSivuProps) => {
         await handleOrganisationMerge(props);
     }
 
-    async function cancelYhdistaOrganisaatio() {
+    function cancelYhdistaOrganisaatio() {
         setYhdistaOrganisaatioModaaliAuki(false);
         setYhdistaOrganisaatio(initialYhdista);
     }
 
+    function handleNimiFromYtj(nimi: Nimi) {
+        if (organisaatioBase) {
+            const newCurrentNimi = nimi;
+            setPerustiedotValue('nimi', newCurrentNimi);
+            const newNimet = [...organisaatioBase.nimet]
+                .sort((a, b) => new Date(a.alkuPvm).getTime() - new Date(b.alkuPvm).getTime())
+                .reverse();
```

Comment:
`Date.before` / `Date.after`?
`Moment.isBefore` / `Moment.isAfter`?
Why `reverse`? Couldn't the sort make proper order in first place?

Reply:
I will fix this to be better implementation